src/init.cpp

```diff
@@ -1272,7 +1272,7 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
     // need to reindex later.
 
     assert(!g_banman);
-    g_banman = std::unique_ptr<BanMan>(new BanMan(GetDataDir() / "banlist.dat", &uiInterface));
+    g_banman = std::unique_ptr<BanMan>(new BanMan(GetDataDir() / "banlist.dat", &uiInterface, DEFAULT_MISBEHAVING_BANTIME));
```

Comment:
I think we need to pass `gArgs.GetArg("-bantime", DEFAULT_MISBEHAVING_BANTIME)` here?  Or else I'm missing how the command line argument gets used after this commit.

Reply:
Very much so, thanks for catching that.